src/CrossSigningManager.js

```diff
@@ -159,6 +162,20 @@ export const crossSigningCallbacks = {
     onSecretRequested,
 };
 
+export async function promptForBackupPassphrase() {
+    let key;
+
+    const RestoreKeyBackupDialog = sdk.getComponent('dialogs.keybackup.RestoreKeyBackupDialog');
+    const { finished } = Modal.createTrackedDialog('Restore Backup', '', RestoreKeyBackupDialog, {
+            showSummary: false, keyCallback: k => key = k,
+    }, null, /* priority = */ false, /* static = */ false);
+
+    const success = await finished;
+    if (!success) throw new Error("Key backup prompt cancelled");
```

Comment:
What is the user experience of throwing an error here...?

Reply:
We need to throw an error so that the bootstrap function knows we failed to gather the key - the bootstrap function will then deal with this accordingly.